test/runtests.jl

```diff
@@ -10,4 +10,12 @@ include(joinpath(pkg_path, "test", "utils.jl"))
 
 @testset "FixedPointDecimals" begin
     include("FixedDecimal.jl")
-end  # global testset
+end
+
+
+if isdefined(Base, Symbol("@assume_effects"))
+@testset "FixedPointDecimals" begin
+    include("fldmod-by-const_tests.jl")
+end
+end
```

Comment:
Why not include this in the existing FixedPointDecimals testset?
```suggestion
    isdefined(Base, Symbol("@assume_effects")) && include("fldmod-by-const_tests.jl")
end
```

Reply:
Oh, oops, I meant to rename the test-set to `fldmod-by-const_tests`.

🤷 i dunno, i guess it seems nice to have a separate test-set per file, so that you can group the failures? but you're right that we probably _also_ want an outer testset so that a failure in one doesn't abort the rest of the tests.. annoying.

I'll take your change, i think that's preferable.